buildscripts/sql-parser-generator.gradle

```diff
@@ -46,6 +46,7 @@ task extractParserTemplate(type: Copy) {
         include 'codegen/templates/Parser.jj'
         include 'codegen/default_config.fmpp'
     }
+    doNotTrackState("test")
```

Comment:
honestly idk what to do with "Failed to create MD5 hash for file content" err without this thing